### ISSUE TITLE
Feature Request: Fixed Workspace Names

### ISSUE THREAD

**Issue by jounathaen** [christopher-l/space-bar#23]
Hi,
Thank you for creating this extension. I also use it to rename the workspaces, but the "Smart Workspaces Names" does not fit my workflow. 
Instead, I prefer a more static approach to workspace names: Workspace 1 is always "Personal", Workspace 2 is always "Work" and the others might change. I could think of two ways of achieving this with this extension:
- Configure a list of names in the extension that are always applied to the respective workspaces (e.g. `["Personal", "Work"]`)
- Remember the names of the workspaces in order, beyond the closing of these or a reboot.

Cheers!

**Comment by christopher-l** [comment]
Hi Jonathan,

Does it work for you to just disable "Smart workspace names". This should
not be needed to rename workspaces and names should be persistent when
disabled.

This has lead to confusion in the past:
https://github.com/christopher-l/space-bar/issues/19

**Comment by jounathaen** [comment]
Hi Christopher,

Thanks for the fast reply. I think it did not, but let me try again. Maybe the reason is, that I have enabled _dynamic workspaces_ as well. Not sure if that works together.

**Comment by jounathaen** [comment]
It seems I was just "holding it wrong" or something. Thank you for your support!
